drivers/include/at86rf2xx.h

```diff
@@ -167,6 +167,11 @@ typedef struct {
     uint8_t pending_tx;                 /**< keep track of pending TX calls
                                              this is required to know when to
                                              return to @ref at86rf2xx_t::idle_state */
+#if defined(MODULE_AT86RF232) || defined(MODULE_AT86RF233)
```

Comment:
IMO the condition here should match the above, i.e., `AT86RF2XX_XAH_CTRL_2__ARET_FRAME_RETRIES_OFFSET`. Just to make sure its consistent and avoid failures in the future.

Reply:
Missed that one :(